build-dpkg/build.sh

```diff
@@ -19,4 +19,6 @@ dpkg-buildpackage $@
 
 ## Move the built artifacts into the Docker mounted workspace
 mkdir -p artifacts
-mv ../*.deb ../*.changes ../*.dsc ../*.buildinfo ../*.tar.gz ./artifacts
+for a in ../*.deb ../*.changes ../*.dsc ../*.buildinfo ../*.ddeb ../*.tar.gz; do
+    [ -f "${a}" ] && mv "${a}" ./artifacts/
```

Comment:
This is redundant. If you've already done glob expansion then you know the file(s) exist. You can drop the `-f` check. Also your original `mv` will actually move all the files that do match, but it will return a failure exit code because of the glob(s) that failed to expand. That makes the whole script return a failure because of the `-e` option.

There are a couple of ways to fix glob so they don't error out like that, but a simple hack is just to or-noop the whole thing:

```
mv ../*.{deb,changes,desc,buildinfo,ddeb,tag.gz} ./artifacts/ ||:
```